src/uu/true/src/true.rs

```diff
@@ -4,16 +4,59 @@
 //  *
 //  * For the full copyright and license information, please view the LICENSE
 //  * file that was distributed with this source code.
+use clap::{App, AppSettings, Arg};
+use std::io::Write;
+use uucore::error::{set_exit_code, UResult};
 
-use clap::{App, AppSettings};
-use uucore::error::UResult;
+static ABOUT: &str = "
+ Returns true, a successful exit status.
+
+ Immediately returns with the exit status `0`, except when invoked with one of the recognized
+ options. In those cases it will try to write the help or version text. Any IO error during this
+ operation causes the program to return `1` instead.
+";
```

Comment:
```suggestion
static ABOUT: &str = "\
Returns true, a successful exit status.

Immediately returns with the exit status `0`, except when invoked with one of the recognized
options. In those cases it will try to write the help or version text. Any IO error during this
operation causes the program to return `1` instead.
";
```